redis/src/types.rs

```diff
@@ -778,13 +778,21 @@ impl fmt::Debug for RedisError {
     }
 }
 
-pub(crate) enum RetryMethod {
+/// What method should be used if retrying this request.
+pub enum RetryMethod {
+    /// Create a fresh connection.
     Reconnect,
+    /// Don't retry, this is a permanent error.
     NoRetry,
+    /// Retry immediately, this doesn't require a wait since it's an IO level issue.
```

Comment:
remove the IO level stuff, or clarify that ti's an example, not every case of this value.